### PR TITLE
EVG-2087 Use workdir with local_files_include_filter

### PR DESCRIPTION
**Pull request by bsamek**

### CODE REVIEW

File: command/s3_put.go [hunk 1]
```diff
@@ -229,10 +239,10 @@ retryLoop:
 		case <-ctx.Done():
 			return errors.New("s3 put operation canceled")
 		case <-timer.C:
-			filesList := []string{s3pc.LocalFile}
+			filesList := []string{s3pc.workDir + s3pc.LocalFile}
```
Comment (tychoish) [inline command/s3_put.go]: filepath.Join()

Comment (bsamek) [reply command/s3_put.go]: fixed

File: command/s3_put.go [hunk 2]
```diff
@@ -229,10 +239,10 @@ retryLoop:
 		case <-ctx.Done():
 			return errors.New("s3 put operation canceled")
 		case <-timer.C:
-			filesList := []string{s3pc.LocalFile}
+			filesList := []string{filepath.Join(s3pc.workDir + s3pc.LocalFile)}
```
Comment (john-m-liu) [inline command/s3_put.go]: the directory and file name should be separate parameters

Comment (bsamek) [reply command/s3_put.go]: sorry, fixed

**Pull request merged**